tools/helmify/src/config.yaml

```diff
@@ -0,0 +1,332 @@
+dex:
+  kustomization_paths:
+    - upstream/common/dex/overlays/istio
+  output_helm_chart_path: charts/common/dex
+  version: 0.1.0
+  app_version: v2.24.0
+
+
+ingress:
+  deployment_options:
+    cognito:
+      kustomization_paths:
+      - awsconfigs/common/istio-ingress/overlays/cognito
+      output_helm_chart_path: charts/common/ingress/cognito
+      version: 0.1.0
+      app_version: v0.1.0
+  params:
+    template_paths:
+      - tools/helmify/template/ingress/params/overlays/cognito/params.env
+      - tools/helmify/template/ingress/params/base/params.env
+    target_paths:
+      - awsconfigs/common/istio-ingress/overlays/cognito/params.env
+      - awsconfigs/common/istio-ingress/base/params.env
+  values:
+    template_paths:
+      - tools/helmify/template/ingress/values/cognito/values.yaml
+    target_paths:
+      - charts/common/ingress/cognito/values.yaml
+
+
+katib: 
+  deployment_options:
+    vanilla:
+      kustomization_paths:
+      - upstream/apps/katib/upstream/installs/katib-with-kubeflow
+      output_helm_chart_path: charts/apps/katib/vanilla
+      version: 0.1.0
+      app_version: v0.14.0-rc.0
+    katib-external-db-with-kubeflow:
+      kustomization_paths:
+      - awsconfigs/apps/katib-external-db-with-kubeflow
+      output_helm_chart_path: charts/apps/katib/katib-external-db-with-kubeflow
+      version: 0.1.0
+      app_version: v0.14.0-rc.0
+      
+
+istio-1-14:
+  kustomization_paths:
+    - upstream/common/istio-1-14/istio-crds/base
+    - upstream/common/istio-1-14/istio-namespace/base
+    - upstream/common/istio-1-14/istio-install/base
+  output_helm_chart_path: ../charts/common/istio-1-14
```

Comment:
Any reason for the ../

Reply:
good catch Ananth. It was a miss, just fixed it .